subsys/zigbee/Kconfig

```diff
@@ -146,7 +146,7 @@ config ZIGBEE_ERROR_TO_STRING_ENABLED
 	  Include functions for mapping ZBOSS errors to string.
 
 # Configure ZBOSS_TRACE_LOG_LEVEL
-choice
+choice ZBOSS_TRACE_LOG_LEVEL_CHOICE
```

Comment:
why is this setting post-fixed with `_CHOICE` when the rest of the changes in this PR is not post-fixed ?

Reply:
It's due to the following `config ZBOSS_TRACE_LOG_LEVEL` (already taken).